js/components/addresssave.js

```diff
@@ -19,50 +20,103 @@ $(document).ready(function() {
 // Takes the autocompleted address, makes a call to Google Maps Geocoder API, 
 // And stores the formatted address string, latitude, and longitude in an 
 // object that is pushed to the addresses array stored in localStorage
-function saveAddress(addressString, label) {
+function saveAddress(addressString, label, isHome) {
     let addressParam = addressString.replace(/ /g,"+");
     let APIURL = "https://maps.googleapis.com/maps/api/geocode/json?address=" + addressParam + "&key=" + GOOGLE_MAPS_KEY;
     $.get(APIURL).done(function(response){
-        storeAddress(response, label);
+        storeAddress(response, label, isHome);
     });
 
 };
 
+// Writes all addresses in the the "address" array to the right side of the page
+
+function writeAddresses() {
+    let addressArray = JSON.parse(localStorage.getItem("addresses"));
+    $("#searchDetails").empty();
+    addressArray.forEach(function(object, index){
+        let icon;
+        let address = object.address.replace(",",",<br>")
+        address = address.replace(", USA", "");
+        if (index === 0) {
+            icon = "home"
+        }
+        else {
+            icon = "place"
+        }
+        $("#searchDetails").append(`
```

Comment:
Using backticks assumes you're going to use template literals, so this can be:

```
$("#searchDetails").append(`
    <li data="${index}">
+                <div class="collapsible-header">
+                    <i class="material-icons">${icon}</i>${object.label}</div>
+                <div class="collapsible-body">
+                    <span>${address}</span>
+                </div>
+            </li>
+        `);
+    });

Reply:
Great suggestion, thanks!